src/components/WorkPoolStatusIcon.vue

```diff
@@ -39,23 +39,7 @@
 
 <style>
 .work-pool-status-icon { @apply
-  flex
-  items-center
   cursor-help
-  w-2
-  h-2
-  align-middle
```

Comment:
Does this even do anything in flex layout? regardless most consuming points that use these icons push their own layouts downward to do vertical alignment when/where necessary.